CMakeLists.txt

```diff
@@ -76,18 +76,17 @@ endif( )
 # BUILD_SHARED_LIBS is a cmake built-in; we make it an explicit option such that it shows in cmake-gui
 option( BUILD_SHARED_LIBS "Build hipBLAS as a shared library" ON )
 
-# Quietly look for CUDA, but if not found it's not an error
-# The presense of hip is not sufficient to determine if we want a rocm or cuda backend
-option(USE_CUDA "Look for CUDA and use that as a backend if found" ON)
-if(USE_CUDA)
-    find_package( CUDA QUIET )
+# Find CUDA if the user wants a CUDA version.
+option(USE_CUDA "Look for CUDA and use that as a backend if found" OFF)
```

Comment:
Maybe I'm mistaken, but does this change not make `USE_CUDA` always OFF? In install.sh we turn it off when the `--cuda` flag is not used, so it's never on.
This failed on a CUDA system for me - it was using the incorrect (rocblas) header file.

Reply:
--cuda in the install.sh script would have to turn in ON now.   The change states OFF will be the default value unless a -DUSE_CUDA=ON which is passed by install.sh, others may not use the install.sh or may use cmake-gui to set options.   You can add a commit for this ?

Reply:
I've pushed the change, didn't realize there was a shell script!